lib/action_controller/serialization.rb

```diff
@@ -27,6 +27,7 @@ def get_serializer(resource, options = {})
       end
       ActiveModel::SerializableResource.serialize(resource, options) do |serializable_resource|
         if serializable_resource.serializer?
+          serializable_resource.url_helper = self
```

Comment:
makes me sad we need to do this :( so much coupling, but I guess that's where we're going.  I coulda sworn you could

```ruby
url = Rails.application.routes.url_for(:controller => "users",

                                           :action => "verify_email",

                                           :id_token => "tok",

                                           :only_path => true)
```